video_xblock/static/js/player_state.js

```diff
@@ -29,18 +29,27 @@ var player_state = {
 var xblockUsageId = window.location.hash.slice(1);
 
 /** Restore default or previously saved player state */
-var setInitialState = function(player, state) {
-  if (state.currentTime > 0) {
-    player.currentTime(state.currentTime);
-  }
-  player
-    .volume(state.volume)
-    .muted(state.muted)
-    .playbackRate(state.playbackRate);
+var setInitialState = function (player, state) {
+    var stateCurrentTime = state.currentTime;
+    var videoXblockState = localStorage.getItem('videoXblockState');
```

Comment:
Current name is very misleading. It's too easy to mess it up with player's `state`.

Please change it to something like `playbackProgress`. So it's clear it stores only the video played videos' progress.